msu/msu_mod/setup_msu_mod.nut

```diff
@@ -1 +1,3 @@
 ::MSU.Mod <- ::MSU.Class.Mod(::MSU.ID, ::MSU.Version, ::MSU.Name);
```

Comment:
These should not be part of this PR because this PR adds the framework. Having it in the PR is probably not a big deal but the way you've got it set up in the commits right now is that this addition happens in the same commit that adds this framework. The addition of MSU links should be a separate commit.